uint32 layer_cnt
string[] layer_names
sensor_msgs/Image[] layers
